lessons/pydata/databases/index.ipynb

```diff
@@ -115,11 +115,11 @@
     "\n",
```

Comment:
<p>Vyměnil bych security → bezpečnost když je všechno ostatní moc hezky česky</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/83/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Přijde mi, že "bezpečnost" je širší pojem, než "security". No ale celkově bych to napsal asi jinak, už ani nevím, co jsem tím přesně myslel :) Zkusím to upravit.</p>